Web/HTML/html5-input-form-attributes/autofocus.md

```diff
@@ -13,24 +13,48 @@ stub: true
 
 tags:
   - deep
-
-
+---
+# Autofocus
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+The HTML input form `autofocus` attribute is used to specify an `<input>` element to have focus once the web page is loaded.
+
+Example:
+```
+<form action="/credit_card.php">
+  Full name: <input type="text"
+    name="fullname" autofocus><br>
+  Card number: <input type="number"
+    name="ccnumber"><br>
+  Exparation date: <input type="month"
+    name="ccdate"><br>
+  CV: <input type"number"
+    name="cvnumber"><br>
+  <input type="submit">
+</form>  
+```
+
+In the example above, we have a credit card input field with the focus set for the `Full name` input field. That means once the page is loaded the user won't have to click on the input field but can rather start typing right away.
+
+You can assign `autofocus` as either `autofocus`, `autofocus=""` or `autofocus="autofocus"`. All three options work the same in HTML5 however,  if you're using XHTML `autofocus` is required to be written as `autofocus="autofocus"`.
+
 
 ---
 ## Practice
 
 Which statements are true regarding autofocus?
 
-+ Automatically sets focus to assigned control when the page loads.
-+ User can start using assigned control without having to select it first.
-+ Can be assiged as: `autofocus` or `autofocus="autofocus"`
-- It should be assigned to every control in the form.
-- Can be applied to text fields, radio buttons, & check boxes. 
-- Additional JS is typically required.
+???
```

Comment:
if the user selects the correct answers in the wrong order, they'll get it wrong - strip it down to one correct t/f answer